echo10-construction/src/echo10_construction.py

```diff
@@ -140,16 +151,21 @@ def create_granule_echo10_in_s3(inputs, config):
     echo10_s3_objects = []
     log.info('Creating echo10 file for %s', inputs['Product']['Key'])
     granule_data = get_granule_data(inputs, config['granule_data'])
+
     echo10_content = render_granule_data_as_echo10(granule_data)
     echo10_s3_object = {
         'bucket': config['output_bucket'],
         'key': granule_data['granule_ur'] + '.echo10',
     }
     echo10_s3_objects.append(echo10_s3_object)
+
     upload_content_to_s3(echo10_s3_object, echo10_content)
 
-    del granule_data['size_mb_data_granule']
-    del granule_data['additional_attributes']['BYTES']
+    if 'size_mb_data_granule' in granule_data:
+        del granule_data['size_mb_data_granule']
+
+    if 'BYTES' in granule_data['additional_attributes']:
+        del granule_data['additional_attributes']['BYTES']
```

Comment:
I'm fine with being defensive here, but did anything in particular prompt the change? Is there a particular test is caused to fail?